template-quality-checklist.md

```diff
@@ -0,0 +1,102 @@
+- [ ] The template set is well-organized and the contents flow in a logical order.
+- [ ] The template is well-written and clear.
+- [ ] The template set provides sufficient guidance about how to fill it in and implement the template in a documentation project.
+
+
+### Raw template requirements - PULL REQUEST REVIEWER ONLY
+
+- [ ] Check rendered Markdown output to ensure it renders correctly and follows the template markdown style guide.
+- [ ] The template title is present and is an H1.
+- [ ] The template includes an introductory comment pointing them to other template resources.
+- [ ] Embedded writing tips are formatted with {curly brackets}.
+- [ ] The template is free from grammar errors and typos.
+- [ ] The scope of the template is appropriate, meaning it is not too simple or overly complex (i.e. it needs to be expanded or broken into multiple smaller templates).
+- [ ] The template is complete and comphrehensive.
+- [ ] The template is well-organized and the contents flow in a logical order.
+- [ ] The template is well-written and clear.
+
+
+### Template guide requirements - PULL REQUEST REVIEWER ONLY
+
+- [ ] Check rendered Markdown output to ensure it renders correctly and follows the template markdown style guide.
+- [ ] The template guide title is present and is an H1.
+- [ ] The template includes an introductory comment pointing them to other template resources.
+- [ ] Embedded writing tips are formatted with {curly brackets}.
+- [ ] The template guide is free from grammar errors and typos.
+- [ ] The template guide includes a "Why do I need this type of document?" section.
+- [ ] The template guide includes a "Contents of this template" section.
+- [ ] The contents listed in the "Contents of this template" section mirror the sections in the raw template.
+- [ ] The template guide includes an implementation checklist (optional).
+- [ ] The template guide includes additional resources (optional).
+- [ ] The template guide is well-organized and the contents flow in a logical order.
+- [ ] The template guide is well-written, clear, and provides sound advice to template users.
+
+
+### Before you start requirements (optional) - PULL REQUEST REVIEWER ONLY
+
+- [ ] Check rendered Markdown output to ensure it renders correctly and follows the template markdown style guide.
+- [ ] The before you start guide title is present and is an H1.
+- [ ] The before you start guide provides helpful advice about necessary requirements before writing this type of documentation.
+- [ ] The template set is well-organized and the contents flow in a logical order.
+- [ ] The template is well-written and clear.
+
+
+### Template example requirements (optional) - PULL REQUEST REVIEWER ONLY
+
+- [ ] Check rendered Markdown output to ensure it renders correctly and follows the template markdown style guide.
+- [ ] The template example provides is well-written and provides a clear example of how to use the template.

```

Comment:
Typo here. Change "provides" to "provided".

Reply:
Sure.

Reply:
I've adopted this suggestion.